src/styles/index.scss

```diff
@@ -1,3 +1,21 @@
+$card-color: #fff;
+$border-color: #f3f3f3;
+$font-main-color: #060b35;
+$font-secondary-color: #616070;
+$inactive-button-color: #00acdc;
+$active-button-color: #fff;
+$inactive-button-font-color: #fff;
+$active-button-font-color: #00acdc;
+$active-button-border-color: #00acdc;
```

Comment:
And let's create a separate file for these variables and import it here

Reply:
I already tried to create a separate file, but after that “npm start” command can’t start browser page, just infinite loading. “npm test” can’t start too…